loaders/translations-loader.js

```diff
@@ -0,0 +1,69 @@
+var loaderUtils = require('loader-utils');
```

Comment:
Can we move /loaders to a /lib folder as well? That way we can document that everything under /lib and /src/lib are part of the scaffold tooling and don't need to be modified.